auth/src/main/java/org/openedx/auth/presentation/preAuth/PreAuthFragment.kt

```diff
@@ -0,0 +1,206 @@
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.tooling.preview.Devices
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.fragment.app.Fragment
+import org.koin.android.ext.android.inject
+import org.openedx.auth.R
+import org.openedx.auth.presentation.AuthRouter
+import org.openedx.core.ui.OpenEdXButton
+import org.openedx.core.ui.OpenEdXOutlinedButton
+import org.openedx.core.ui.SearchBar
+import org.openedx.core.ui.displayCutoutForLandscape
+import org.openedx.core.ui.noRippleClickable
+import org.openedx.core.ui.theme.OpenEdXTheme
+import org.openedx.core.ui.theme.appColors
+import org.openedx.core.ui.theme.appTypography
+
+class PreAuthFragment : Fragment() {
+
+    private val router: AuthRouter by inject()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ) = ComposeView(requireContext()).apply {
+        setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+        setContent {
+            OpenEdXTheme {
+                PreLoginScreen(
+                    onSignInClick = {
+                        router.navigateToSignIn(parentFragmentManager)
+                    },
+                    onRegisterClick = {
+                        router.navigateToSignUp(parentFragmentManager)
+                    },
+                    onSearchClick = { querySearch ->
+                        router.navigateToDiscoverCourses(parentFragmentManager, querySearch)
+                    }
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun PreLoginScreen(
+    onSearchClick: (query: String) -> Unit,
+    onRegisterClick: () -> Unit,
+    onSignInClick: () -> Unit,
+) {
+
+    var textFieldValue by rememberSaveable(stateSaver = TextFieldValue.Saver) {
+        mutableStateOf(TextFieldValue(""))
+    }
+    val scaffoldState = rememberScaffoldState()
+    val scrollState = rememberScrollState()
+    Scaffold(
+        scaffoldState = scaffoldState,
+        modifier = Modifier
+            .fillMaxSize(),
+        backgroundColor = MaterialTheme.appColors.background
+    ) {
+        Surface(
+            modifier = Modifier
+                .padding(it)
+                .fillMaxSize()
+                .verticalScroll(scrollState)
+                .displayCutoutForLandscape(),
+            color = MaterialTheme.appColors.background
+        ) {
+            Column(
+                modifier = Modifier.padding(
+                    start = 16.dp,
+                    end = 16.dp,
+                    top = 32.dp,
+                    bottom = 32.dp
+                )
+            ) {
+                Image(
+                    painter = painterResource(id = org.openedx.core.R.drawable.core_ic_logo),
```

Comment:
move this to import, for example 
`import org.openedx.core.R as CR`

Reply:
updated with `coreR`